source/material_model/rheology/visco_plastic.cc

```diff
@@ -497,6 +499,11 @@ namespace aspect
                            "dynamic stresses are much higher than the lithostatic pressure. "
                            "If false, the minimum pressure in the plasticity formulation will "
                            "be set to zero.");
+        prm.declare_entry ("List of component masks", "",
```

Comment:
I am not able to understand what this does from the parameter name and description. Can you try to improve this?

I assume this is currently "list of compositional field names to ignore"? I would prefer to negate this as "list of names of compositional fields to use" (and treat the empty case as meaning "all").